tests/Unit/CrudPanel/CrudPanelReadTest.php

```diff
@@ -294,4 +294,61 @@ public function testHasUploadFieldsUpdateFormUnknownId()
         $unknownId = DB::getPdo()->lastInsertId() + 1;
         $this->crudPanel->hasUploadFields('update', $unknownId);
     }
+
+    public function testEnableDetailsRow()
+    {
+        $this->crudPanel->enableDetailsRow();
+
+        $this->assertTrue($this->crudPanel->details_row);
+    }
+
+    public function testDisableDetailsRow()
+    {
+        $this->crudPanel->disableDetailsRow();
+
+        $this->assertFalse($this->crudPanel->details_row);
+    }
+
+    public function testSetDefaultPageLength()
+    {
+        $pageLength = 20;
+        $this->crudPanel->setDefaultPageLength($pageLength);
+
+        $this->assertEquals($pageLength, $this->crudPanel->getDefaultPageLength());
+    }
+
+    public function testGetDefaultPageLength()
+    {
+        $defaultPageLength = $this->crudPanel->getDefaultPageLength();
+
+        $this->assertEquals(25, $defaultPageLength);
```

Comment:
Is that 25 hard coded in the actual code?

Reply:
yup.